src/plugins/timeline/public/directives/timeline_help/timeline_help.html

```diff
@@ -260,8 +260,8 @@ <h4 i18n-id="timeline.help.querying.countTitle" i18n-default-message="Beyond cou
   </div>
 
   <div ng-show="page === 4">
-    <div>)
-      <h2 i18n-id="timeline.help.expressionsTitle" i18n-default-message="Expressing yourself with expressions"></h2>
+    <div>
+      <h2 i18n-id="timeline.help.expressionsTitle" i18n-default-message="Expressing ROCKY with expressions"></h2>
```

Comment:
Does it need to be `ROCKY` ?

Reply:
Good catch! Sorry, I was trying to verify that my changes were being applied. Will revert.